.github/workflows/swap-azure-web-app-slots.yml

```diff
@@ -118,6 +145,80 @@ jobs:
             -WebAppName ${{ inputs.app-name }} `
             -SlotName ${{ inputs.destination-slot-name }}
 
+      - name: Checkout Repository
+        if: inputs.swap-code-repo != ''
+        uses: Lombiq/GitHub-Actions/.github/actions/checkout@dev
+        with:
+          repository: ${{ inputs.swap-code-repo }}
+          token: ${{ secrets.TAG_DEPLOYMENT_TOKEN }}
+
+      - name: Setup Git Config
+        if: inputs.swap-code-repo != ''
```

Comment:
These could also be a single step, not requiring repeating this `if`.

Reply:
Still this can be merged at least with Fetch tags. But the latter isn't actually needed, since it's done in `remove-old-tags`.